index.src.html

```diff
@@ -757,8 +757,22 @@ <h3 id="try-delivery" algorithm>
 
 <section>
   <h2 id="observers">Reporting Observers</h2>
-  Each <a>environment settings object</a> has an associated list of
-  <dfn>registered reporting observers</dfn> ({{ReportingObserver}} objects).
+
+  A <dfn>reporting observer</dfn> observes some types of <a>reports</a> from
+  JavaScript, and is represented in JavaScript by the {{ReportingObserver}}
+  object.
+
+  Each <a>environment settings object</a> has a <dfn>registered reporting
+  observer list</dfn>, which is a list of <a>reporting observers</a>.
+
+  To <dfn>register</dfn> a <a>reporting observer</a> means to add it to
+  the <a>registered reporting observer list</a>.
+
+  Each <a>environment settings object</a> has a <dfn>report buffer</dfn>, which
+  is an ordered list of <a>reports</a> that have been generated in
```

Comment:
You don't say where things get added to the report buffer, or how overflow is handled.  I think that should just be a matter of adding a step to the 'notify' algorithm.  I think other observer specs have a pattern you can follow?

Reply:
I have added to the notify algorithm how the reports get added to the report buffer.